### PR TITLE
Create Foreword section and move content from Welcome page

### LINKED ISSUE #2033: Foreword section in the book
### Summary

* With the growing resources, it is needed to make community and project norms more explicit and hence provide more background and policy in the foreword of the book
* This issue builds on #1777

### What needs to be done?

- [ ] Create a foreword section (similar to afterword)
- [ ] Add the following documents
  - [ ] Vision and mission
  - [ ] Background and history + community health report
  - [ ] Turing Specific jargons (See discussions #1961)
  - [ ] Governance overview (see #2035)

### Who can help?

* Core contributors of the project including those who share an interest in helping us do some community research in The Turing Way

---

### Updates

-

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #2033 

* This PR creates a foreword section with project info, making welcome page light weight and easy to browse

### List of changes proposed in this PR (pull-request)

- [x] Revamp welcome page
- [x] Add foreword section with the following chapters
  - [x] Vision and Mission
  - [x] Background and History
  - [x] Navigating the book
  - [x] Community and Roles
  - [x] Ways of Working
  - [x] Impact
  - [x] Cite us

### What should a reviewer concentrate their feedback on?

- [ ] What other section should be added
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@glimmerglen tagging you to this PR.

**Comment by aleesteele** [comment]
Adding myself to this PR, as I'm reading through things with a few things in mind:
- How would a newcomer to the project interact with this page, and the phrasing therein?
- What information do different personas prioritise within the project?
- What is clear/unclear for these personas?

File: book/website/foreword/navigate.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* {ref}`pd`
+* {ref}`cm`
+* {ref}`cl`
+* {ref}`er`
+* {ref}`ch`
+
+```{figure} ../figures/theturingway-chapters.jpg
+---
+name: theturingway-chapters
+alt: The Turing Way Guide to reproducible research and it's stucture illustrated to show a set of doors to represent how it's built on chapters and sub chapters of the different areas of the guide
```
Comment (aleesteele) [inline book/website/foreword/navigate.md:16]:
```suggestion
alt: The Turing Way Guide to reproducible research and its structure illustrated to show a set of doors to represent how it's built on chapters and sub chapters of the different areas of the guide
```

File: book/website/foreword/impact.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Impact Story
+
+We provide an overview of the impact story featured by The Alan Turing Institute in 2020. 
+Please read the full report online written by Sean O'Neil through an interview with the project lead, Kirstie Whitaker: [https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science](https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science).
+
+- There is a crisis of reproducibility  in science. 
+- ‘Publish-or-perish’ incentives and excessive data secrecy stifles progress and wastes resources: siloed science is slow science. 
+- _The Turing Way_ is an evolving online ‘handbook’ on how to conduct world-leading, reproducible research in academic data science and AI with the goal of making reproducible research “too easy not to do”.  
+- The book deals not only with the ‘how’ of things, but also the ‘why’ – the ethos and long-term benefits of reproducible research.
+- _The Turing Way_ is also a flourishing global community of research engineers, data librarians, industry professionals and research experts dedicated to capturing and sharing research best-practise, tools and data.
```
Comment (aleesteele) [inline book/website/foreword/impact.md:18]:
```suggestion
- _The Turing Way_ is also a global community of research engineers, data librarians, industry professionals and research experts dedicated to capturing and sharing research best practices, tools and data for data science.  
```

File: book/website/foreword/navigate.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+---
+_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300.
+```
+
+Each guides contain multiple chapters, and each chapters are divided into smaller, self-contained subchapters to allow you to naviagte the book and its content easily in your own pace
```
Comment (aleesteele) [inline book/website/foreword/navigate.md:21]:
```suggestion
Each guides contain multiple chapters, and each chapters are divided into smaller, self-contained sub-chapters to allow you to navigate the book and its content easily in your own pace. Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first. 

**Remember: _The Turing Way_ is not meant to be read from beginning to end!**
```

**Review by aleesteele — APPROVED** [review]
Adding comments for review in progress

File: book/website/foreword/foreword.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+_The Turing Way_ is open for anyone to use, contribute to and share. 
+You can inspect our resources on [GitHub](https://github.com/alan-turing-institute/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
+
+Although _The Turing Way_ core team is located in London (UK) and receives support from [The Alan Turing Institute](https://www.turing.ac.uk/), the project is designed to be a global collaboration.
+Chapters have been written, reviewed and curated by members from international research institutes and universities, government departments, and industry.
+
+The foreword section of _The Turing Way_ handbook provides the following information:
+
+- {ref}`Vision and Mission<fw-vision-mission>` 
+- {ref}`Background and history<fw-background>`
```
Comment (aleesteele) [inline book/website/foreword/foreword.md:13]:
```suggestion
- {ref}`Background and history<fw-history>`
```

File: book/website/welcome.md [hunk 5]
```diff
@@ -47,46 +70,90 @@ To join this community, please read our [contribution guidelines](https://github
...
 
-(welcome-history)=
-## History
+(team-community)=
+## _The Turing Way_ Team and Community
+
+_The Turing Way_ is developed, maintained and driven by our wonderful {ref}`community of contributors<contributors-record-highlights>`.
 
-The book started in January 2019 as a guide for reproducibility, covering {ref}`rr-vcs`, {ref}`rr-testing`, and {ref}`rr-ci`. However, technical skills are just one aspect of making data science research "open for all" and so in February 2020, _The Turing Way_ expanded into a series of guides: {ref}`rr`, {ref}`pd`, {ref}`cm`, {ref}`cl`, and {ref}`er`.
+_The Turing Way_ project receives support and funding from [The Alan Turing Institute](https://www.turing.ac.uk/) and is hosted under the [Tools, Practices and Systems programme](https://www.turing.ac.uk/research/research-programmes/tools-practices-and-systems) led by the project lead **Kirstie Whitaker**.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/welcome.md:119]:
```suggestion
Hosted under the [Tools, Practices and Systems Research Programme](https://www.turing.ac.uk/research/research-programmes/tools-practices-and-systems), _The Turing Way_ project receives support and funding from [The Alan Turing Institute](https://www.turing.ac.uk/) and is hosted under the [Tools, Practices and Systems programme](https://www.turing.ac.uk/research/research-programmes/tools-practices-and-systems) and co-led by **Drs. Kirstie Whitaker** and **Malvika Sharan**.
```

**Comment by aleesteele** [comment]
Tied to #3159

**Comment by aleesteele** [comment]
Bringing some of these into #3159 to start the forward section!

**Comment by aleesteele** [comment]
Adding this here as a review for this foreword - will tackle this during the week of 26 Feb: https://docs.google.com/document/d/1Z_r0G6dmtlw0icQ_ctkhLG_QxZDhuRZaSTSkGvAVK6w/edit

**Review by jessieontherun — CHANGES_REQUESTED** [review]

File: book/website/foreword/community-roles.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+Some of these workshops has become part of regular events and conferences to introduce _The Turing Way_ resources and data science practices among a new audience and early career researcher groups. 
+
+Delivery of these workshops includes the following roles:
+
+* **Expert facilitators and trainers** of Open Science, Reproducibility and ethics-related workshops, both organised by _The Turing Way_ and delivered in external events
+* **Technical trainer** for novice learners on tools like GitHub, Binder, Jupyter and Literate Programming 
+* **Presenters and speakers** for international events/conferences who represent _The Turing Way_ in external platforms
+
+All members are supported to structure their work in _The Turing Way_ to make them findable and onboard new members in their work to provide continuity of their work.
+Researchers are also supported to lead community-based discussions and collaborations, that lead to projects and outputs beyond _The Turing Way_.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/foreword/community-roles.md]: Integrated changes manually - thank you @jessieontherun.

File: book/website/foreword/community-roles.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+For instance, _The Turing Way_ team and community members have been workshops from the very beginning.
+Some of these workshops has become part of regular events and conferences to introduce _The Turing Way_ resources and data science practices among a new audience and early career researcher groups. 
+
+Delivery of these workshops includes the following roles:
+
+* **Expert facilitators and trainers** of Open Science, Reproducibility and ethics-related workshops, both organised by _The Turing Way_ and delivered in external events
+* **Technical trainer** for novice learners on tools like GitHub, Binder, Jupyter and Literate Programming 
+* **Presenters and speakers** for international events/conferences who represent _The Turing Way_ in external platforms
+
+All members are supported to structure their work in _The Turing Way_ to make them findable and onboard new members in their work to provide continuity of their work.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/foreword/community-roles.md]: Integrated changes manually - thank you @jessieontherun.

File: book/website/foreword/community-roles.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+Community members have been taking roles like trainers, workshop leads, translators, infrastructure maintenance, accessibility experts, representatives of different projects in _The Turing Way_ and advocates of different topics of _The Turing Way_'s interest in external projects.
+Some of these areas of work have recently been formalised and are also being led by volunteer members (see Governance Roles).
+For instance, _The Turing Way_ team and community members have been workshops from the very beginning.
+Some of these workshops has become part of regular events and conferences to introduce _The Turing Way_ resources and data science practices among a new audience and early career researcher groups. 
+
+Delivery of these workshops includes the following roles:
+
+* **Expert facilitators and trainers** of Open Science, Reproducibility and ethics-related workshops, both organised by _The Turing Way_ and delivered in external events
+* **Technical trainer** for novice learners on tools like GitHub, Binder, Jupyter and Literate Programming 
+* **Presenters and speakers** for international events/conferences who represent _The Turing Way_ in external platforms
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/foreword/community-roles.md]: Integrated changes manually - thank you @jessieontherun.

File: book/website/foreword/community-roles.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+They are also given mentorship opportunities in the project as they make their contributions to _The Turing Way_ or other open source projects and are encouraged to mentor new participants of the project. 
+
+Community members have been taking roles like trainers, workshop leads, translators, infrastructure maintenance, accessibility experts, representatives of different projects in _The Turing Way_ and advocates of different topics of _The Turing Way_'s interest in external projects.
+Some of these areas of work have recently been formalised and are also being led by volunteer members (see Governance Roles).
+For instance, _The Turing Way_ team and community members have been workshops from the very beginning.
+Some of these workshops has become part of regular events and conferences to introduce _The Turing Way_ resources and data science practices among a new audience and early career researcher groups. 
+
+Delivery of these workshops includes the following roles:
+
+* **Expert facilitators and trainers** of Open Science, Reproducibility and ethics-related workshops, both organised by _The Turing Way_ and delivered in external events
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/foreword/community-roles.md]: Integrated changes manually - thank you @jessieontherun.

File: book/website/welcome.md [hunk 10]
```diff
@@ -3,90 +3,140 @@
...
+**[Tips for users and participants](foreword/join-us)** 💡
+^^^
+**Top tip for the users**: 
+❎ _The Turing Way_ is not meant to be read from start to finish.
+✅ Browse the different guides that you are interested in, or use the search box to find chapters on topics you would like to learn about first.
+
+**Top tip for the participants**: We provide multiple entry points for you to start contributing based on your interest, availability or skill requirements.
+Please {ref}`fw-join-us` in _The Turing Way_.
+
+For all processes and workflows for community participation and involvements, please see the {ref}`ch`.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/welcome.md]: Integrated changes manually - thank you @jessieontherun.

File: book/website/welcome.md [hunk 11]
```diff
@@ -3,90 +3,140 @@
...
-We are committed to creating a space where people with diverse expertise and lived experiences can share their knowledge with others to allow us all to use data science to improve the world.
+**[Who should use _The Turing Way_?](foreword/our-community)** 👋
+^^^
+**All stakeholders in research**, including funders, policymakers and public sectors are encouraged to use _The Turing Way_ to understand their roles and responsibility of reproducibility in data science.
 
-We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
-Accordingly, everyone who participates in _The Turing Way_ project is expected to show respect and courtesy to other community members at all times.
-All contributions must abide by our [code of conduct](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE_OF_CONDUCT.md).
+Our audience includes, but are not limited to, undergrad students, PhD researchers, postdocs, software engineers, Principle Investigators, project coordinators, community managers, data stewards, librarians, data science educators and other roles in research.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/welcome.md]: Added that in the welcome page

**Comment by malvikasharan** [comment]
Thank you @jessieontherun - Integrated the changes. I am going to merge this with a note that more people will review this. Exciting!!

**Pull request merged**